src/frontend/src/eth/services/erc721.services.ts

```diff
@@ -106,7 +108,10 @@ const loadCustomTokensWithMetadata = async (
 						standard: 'erc721' as const,
 						category: 'custom' as const,
 						enabled,
-						version
+						version,
+						...(nonNullish(section) && {
+							section: 'Spam' in section ? CustomTokenSection.SPAM : CustomTokenSection.HIDDEN
```

Comment:
same (P.S. you could create a small util `mapTokenSection`, since it is already used twice and it is an hard-coded value)

```suggestion
							section: section === 'Spam' ? CustomTokenSection.SPAM : CustomTokenSection.HIDDEN
```